### ISSUE TITLE
Warning: Can't find file "MachDeps.h" in directories GHC

### ISSUE THREAD

**Issue by Kleidukos** [ndmitchell/hlint#1074]
* Hlint version: v3.1.6
* OS: Fedora 32
* Repo: The GHC namespace in [`base`](https://gitlab.haskell.org/ghc/ghc/-/tree/master/libraries/base/GHC)

```haskell

Warning: Can't find file "MachDeps.h" in directories
	GHC
	.
  Asked for by: GHC/Enum.hs  at line 23 col 1
hlint: #error Unhandled value for WORD_SIZE_IN_BITS
in GHC/Enum.hs  at line 672 col 1
CallStack (from HasCallStack):
  error, called at ./Language/Preprocessor/Cpphs/CppIfdef.hs:113:21 in cpphs-1.20.8-DevT7HkEKU3DDQgUoHVnJY:Language.Preprocessor.Cpphs.CppIfdef
 ```

It feels related to https://github.com/ndmitchell/hlint/issues/228

**Comment by shayne-fletcher** [comment]
Hi @Kleidukos , I see you've closed this out. I have a bunch of questions 😀 :
  - Is it closed because you've determined it's not a HLint problem?
    - If so, what was the problem and what was it's resolution?
  - Is HLint in general working OK on Fedora 32?
  - Are you using HLint with ghc-8.10.1?
      - If so, out of interest, have you tried with an older GHC?

**Comment by Kleidukos** [comment]
Hi @shayne-fletcher ! The real issue I had is actually on https://github.com/ndmitchell/hlint/issues/1075

> Is it closed because you've determined it's not a HLint problem? 
'tis not! I had forgotten to add the CPP include paths

> Is HLint in general working OK on Fedora 32?
Like a charm!

> Are you using HLint with ghc-8.10.1? 
It's compiled with lts-14.20, so 8.6.5.

**Comment by shayne-fletcher** [comment]
> > Is HLint in general working OK on Fedora 32?
> 
> Like a charm!

This is very satisfying to hear 😍 .
 
> > Are you using HLint with ghc-8.10.1?
> 
> It's compiled with lts-14.20, so 8.6.5.

Quite the stress test then as it will be using `ghc-lib`!

**Comment by Kleidukos** [comment]
@shayne-fletcher Do you have any wish regarding how I'm compiling it while working on GHC? (we take this discussion elsewhere, by the way)

**Comment by shayne-fletcher** [comment]
> @shayne-fletcher Do you have any wish regarding how I'm compiling it while working on GHC? (we take this discussion elsewhere, by the way)

No, it's all good!
